presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaSplitManager.java

```diff
@@ -129,6 +138,22 @@ public ConnectorSplitSource getSplits(ConnectorTransactionHandle transaction, Co
         }
     }
 
+    private static String readSchema(String dataSchemaLocation)
+    {
+        try {
+            InputStream inputStream = isURL(dataSchemaLocation.trim().toLowerCase(Locale.ENGLISH)) ? new URL(dataSchemaLocation).openStream() : new FileInputStream(new File(dataSchemaLocation));
```

Comment:
- static import `ENGLISH`
- `new FileInputStream(dataSchemaLocation)` is more concise
- add `e` as the cause to the `PrestoException` on L148